apps/reactotron-app/src/renderer/KeybindHandler.tsx

```diff
@@ -127,13 +134,22 @@ function KeybindHandler({ children }) {
     ToggleSidebar: () => {
       toggleSideBar()
     },
+    ToggleSearch: () => {
+      // If we're on the timeline page, toggle the search, otherwise switch to the timeline tab and open search
+      if (window.location.hash === "#/") {
+        toggleSearch()
+      } else {
+        openSearch()
+        handlers.OpenTimelineTab()
+      }
+    },
```

Comment:
if a user is on a different page, we **open** the search, instead of `toggleSearch`. That way it's always open when going to the timeline, and when a user is on the timeline, they can `toggleSearch`